onnxscript/rewriter/onnxruntime/transformers/multihead_attention.py

```diff
@@ -109,7 +109,12 @@ def infer_attn_size_config(self, function: ir.Function) -> AttnSizeConfig:
                     constant_node.op_type == "Constant"
                 ), "Expected the second input to Reshape to be a Constant node."
                 value = _ir_utils.propagate_const_value(reshape_node.inputs[1])
-                constant_numpy_value = _ir_utils.get_numpy_from_ir_value(value)
+                constant_value = value.const_value
+                if constant_value is None:
+                    raise function_rule.FunctionRewriteError(
```

Comment:
Will this fail the whole thing or is it captured?

Reply:
Captured. The rule would be treated as no matched.